packages/atomizer/bin/atomizer

```diff
@@ -204,14 +225,23 @@ if (typeof programOpts.helpersNamespace !== 'undefined') {
     options.helpersNamespace = programOpts.helpersNamespace;
 }
 
-if (programOpts.watch && programOpts.watch.length) {
+// Setup Watcher
+if (programOpts.watch === true || programOpts.watch.length) {
```

Comment:
The main change for the `--watch` option which will now automatically watch for the files parsed. But still allows explicit watching as before.